package.json

```diff
@@ -5,21 +5,28 @@
   "private": true,
   "scripts": {
     "build": "npm run build:production",
-    "build:dev": "npm run remove-dist && npm run build:static && webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo",
-    "build:production": "npm run remove-dist && npm run build:static && webpack --mode=production",
+    "prebuild:dev": "npm run remove-dist",
+    "build:dev": "webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo",
+    "prebuild:production": "npm run remove-dist",
+    "build:production": "webpack --mode=production",
     "build:test": "npm run build:production -- --include-tests",
-    "build:static": "gulp imagemin",
+    "prebuild:storybook": "npm run remove-dist --silent",
```

Comment:
`--silent` is an NPM flag that suppresses NPM related output. It helps to avoid displaying unnecessary information generated by a subcommand.

For example, this is what you see when you don't use `--silent` for a subcommand:

![without--silent](https://user-images.githubusercontent.com/873049/96708688-40ef8200-13a2-11eb-9aa9-c847d910ccf6.png)

And here is how it looks like when you use `--silent` for a subcommand:

![with--silent](https://user-images.githubusercontent.com/873049/96708745-58c70600-13a2-11eb-8792-44d1377b73be.png)

Also it worth mentioning that `--silent` doesn't affect non-NPM related output, so storybook logs will be shown even if we use `--silent` flag.

Reply:
Let's not add the `--silent` flags for now. I agree it's a bit cleaner output but unless there's a reason why this is important to do I'd prefer we don't hide the commands that run by default. We can always pass `--silent` to the command itself at run time.

Reply:
Updated.